src/sentry/services/hybrid_cloud/user/__init__.py

```diff
@@ -147,24 +176,24 @@ def get_user(self, user_id: int) -> Optional[APIUser]:
         A user ID to fetch
         :return:
         """
-        users = self.get_many([user_id])
+        users = self.get_many(filter=dict(user_ids=[user_id]))
         if len(users) > 0:
             return users[0]
         else:
             return None
 
-    @abstractmethod
-    def query_users(
-        self,
-        user_ids: Optional[List[int]] = None,
-        is_active: Optional[bool] = None,
-        organization_id: Optional[int] = None,
-        project_ids: Optional[List[int]] = None,
-        team_ids: Optional[List[int]] = None,
-        is_active_memberteam: Optional[bool] = None,
-        emails: Optional[List[str]] = None,
-    ) -> List[User]:
-        pass
+    # @abstractmethod
+    # def query_users(
+    #     self,
+    #     user_ids: Optional[List[int]] = None,
+    #     is_active: Optional[bool] = None,
+    #     organization_id: Optional[int] = None,
+    #     project_ids: Optional[List[int]] = None,
+    #     team_ids: Optional[List[int]] = None,
+    #     is_active_memberteam: Optional[bool] = None,
+    #     emails: Optional[List[str]] = None,
+    # ) -> List[User]:
+    #     pass
```

Comment:
Should we delete these instead of commenting them out?

Reply:
totally, this entire PR is just some scratch work demonstrating the interfaces. It needs a lot of work before it's merged